applications/minotari_ledger_wallet/wallet/src/main.rs

```diff
@@ -144,9 +152,13 @@ impl KeyType {
     fn from_branch_key(n: u64) -> Result<Self, AppSW> {
         // These numbers need to match the TransactionKeyManagerBranches in:
         // base_layer/core/src/transactions/key_manager/interface.rs
+        // See `pub enum TransactionKeyManagerBranch` in
+        // `tari_wallet/src/transaction_service/transaction_key_manager.rs` for the mapping of the branch key to
+        // the key type.
         match n {
-            7 => Ok(Self::Spend),
             6 => Ok(Self::OneSidedSenderOffset),
+            7 => Ok(Self::Spend),
```

Comment:
Is this correct?
I dont think spend should be here?

Reply:
It comes from PR #6413